use-case-guides/workday-request-time-off/README.md

```diff
@@ -209,52 +178,49 @@ curl --location 'https://{{domain}}.workday.com/ccx/api/absenceManagement/v1/{{i
 
-### **Step 3: Build in Creator Studio**
+## Step 3: Build in Creator Studio
 
-We will be building this use case as a query triggered path along with an iPaaS to chain APIs 3 and 4 together in order to submit a time off request for a user. We will be following the P1: Sequential Action Chaining design pattern in your iPaaS to build this use case
+We will be building this plugin as a Path along with an iPaaS to chain APIs 3 and 4 together in order to submit a time off request for a user.
 
-### Design Pattern P1: Sequential Action Chaining
+### Plugin Architecture Diagram
 
-![Untitled](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Untitled.png)
+![Untitled](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Untitled.png)
 
-✨ **Triggers**
+**Triggers**
 
 1. Natural language input to copilot requesting to take time off
 
-**🗣️ Natural Language Slots**
+**Natural Language Slots**
 
 1. Start Date: when time off starts
 2. End Date *(optional)*: when time off ends. By default end date is the same as the start date.
 3. Comments *(optional)*: Any relevant context / justification for taking time off
 4. Start Time *(optional)*: if taking partial days off, starting time for time off can be specified.
 5. End Time *(optional)*: if taking partial days off, ending time for time off can be specified.
 
-🤲 **Inferred Slots**
-
-1. Current Date: used in the "as of date" in the request PTO action. Could be static.
-2. Email Address: inferred from user roster profile of the employee calling this plugin
-
-**🗺️ Resolvers**
-
-1. Time Off Plan: maps an employee’s time off plans to the corresponding time off types (which is required by the request time off API)
-
-**🏃‍♂️ Actions**
+**Actions**
 
-1. Lookup time off type details: Executes the `/api/wql/v1/{{instance}}/data` API to query the requesting employee's workday ID, eligible time off plans, and relevant time off types through their work email address.
+1. Lookup time off type details: Executes the `/api/wql/v1/{{instance}}/data` API to query the requesting employee's Workday ID, eligible time off plans, and relevant time off types through their work email address.
 2. Request time off: Executes the `/api/absenceManagement/v1/{{instance}}/workers/{{worker_id}}/requestTimeOff` API to request time off
 
-**📚 Guidelines**
-
-1. (before this plugin): Employee looks up available time off balance for a particular date, specifies which time off plan they want to use for time off. 
-2. Employee specifies the start date for time off, and optionally the end date, comments, start / end times to their copilot
-3. The time off request is submitted to the employee's manager for approval
-
-**⚠️ Caveats**
+**Caveats**
 
 1. Assuming an employees select a time off plan that has adequate time off balance. An error message is thrown for an invalid time off 
```

Comment:
How will they do this?

Reply:
more explicitly clarified how this works